dashboard/src/modules/components/TableOfContent/index.jsx

```diff
@@ -70,6 +70,9 @@ const TableOfContent = () => {
   const setTableData = (data) => {
     dispatch(updateTableData(data));
   };
+  const setContnetData = (data) => {
```

Comment:
Was this supposed to be `setContentData`??

Reply:
I'd never thought of IDEs as _enabling_ misspellings before....  🤯